### DISCUSSION TITLE
GoDNS FreeBSD port?

### DISCUSSION THREAD

**Discussion by johndo100** [TimothyYe/godns#206] [Ideas]
Hi,

I was happy with GoDNS on my Linux.
I switched to FreeBSD recently.
The FreeBSD binary file is available but I haven't try it yet. The FreeBSD port is more convenient.
I'm trying to learn how to make a FreeBSD port from [here](https://docs.freebsd.org/en/books/porters-handbook/new-port/).
Anyone can share your experience on FreeBSD port?

**Comment by TimothyYe** [comment]
There is already a FreeBSD port: https://www.freshports.org/dns/godns
